### PR TITLE
Feedback fixes

### PR DESCRIPTION
**Pull request by tatianainama**
Summary:
  - Add `title` to sidebar links
  - Show sublist icon while hovering categories (sidebar)
  - Fix feedback form title
  - Fix label's sibling font weight
  - *Bonus Introduce `ImageShowcase` component to allow teams to display inline images (no more tables!)*

### CODE REVIEW

File: docusaurus/src/theme/DocSidebar.js [hunk 1]
```diff
@@ -8,15 +8,22 @@ const WEB_LINKS = [
   [<>© Stream.IO, Inc. <br/> All Rights Reserved.</>, '/']
 ]
 
+const addTitle = (sidebarItems) =>
+  sidebarItems.map(({ label, items, ...props }) => ({ 
+    ...props,
+    ...(items && { items: addTitle(items) }),
+    label: <span title={label}>{label}</span>
```
Comment (tsirlucas) [inline docusaurus/src/theme/DocSidebar.js:15]: why is this needed?

Comment (tatianainama) [reply docusaurus/src/theme/DocSidebar.js:15]: To add the `title` property to links so browsers will show up tooltip when hovering on those links.
The default `DocSidebar` doesn't allow custom props for items **but** it let us use html inside labels

**Review by tsirlucas — APPROVED** [review]
